hydra-node/test/Hydra/OptionsSpec.hs

```diff
@@ -26,6 +29,21 @@ spec = parallel $
   describe "Hydra Node RunOptions" $ do
     -- NOTE: --node-id flag needs to be set so we set a default here
     let setFlags a = ["--node-id", "node-id-1"] <> a
+    it ("validateRunOptions: using more than " <> show (maximumNumberOfParties + 1) <> " parties should error out") $ do
```

Comment:
Using more than `maximumNumerOfParties` should not be allowed. The +1 needs to be on the number of command line argument side.